### PR TITLE
Added two recent talks to README.md

### PR DESCRIPTION
**Pull request by hvaara**
Added the [RocksDB meetup](https://www.youtube.com/watch?v=-ij2OiDTxz0) and [Code Driven NYC](https://www.youtube.com/watch?v=tV-WXM2IJ3U) talks to the readme. Don't know if it's an exhaustive list and if you want this added or not.

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4143)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,7 +155,11 @@ Once that's done, take a look at our [open issues](https://github.com/cockroachd
 
 ## Talks
 
-* [Venue: Golang UK Conference 2015](https://www.youtube.com/watch?v=33oqpLmQ3LE), by [Ben Darnell](https://github.com/bdarnell) on (09/10/2015), 52min.<br />
+* [Venue: Annual RocksDB meetup at Facebook HQ](https://www.youtube.com/watch?v=-ij2OiDTxz0), by [Spencer Kimball] (https://github.com/spencerkimball) on (12/02/2015), 21min.<br />
+  CockroachDB's MVCC model.
+* [Venue: Code Driven NYC](https://www.youtube.com/watch?v=tV-WXM2IJ3U), by [Spencer Kimball] (https://github.com/spencerkimball) on (10/28/2015), 30min.<br />
+  Architecture & Overview.
+* [Venue: Golang UK Conference 2015](https://www.youtube.com/watch?v=33oqpLmQ3LE), by [Ben Darnell](https://github.com/bdarnell) on (08/21/2015), 52min.<br />
```
Comment (tamird) [inline README.md]: was this date previously wrong?

Comment (hvaara) [inline README.md]: It was the date the video was published on YouTube, not the date the talk was held. (Sep 10 vs Aug 21).

Comment (tamird) [inline README.md]: Cool, thanks for fixing that!

Comment (hvaara) [inline README.md]: You're welcome. Thanks for merging! :smile:

**Pull request merged**